eZ/Publish/Core/Repository/ContentTypeService.php

```diff
@@ -934,11 +934,14 @@ public function loadContentTypeList(array $contentTypeIds, array $prioritizedLan
     /**
      * {@inheritdoc}
      */
-    public function loadContentTypes(APIContentTypeGroup $contentTypeGroup, array $prioritizedLanguages = [])
-    {
+    public function loadContentTypes(
+        APIContentTypeGroup $contentTypeGroup,
+        array $prioritizedLanguages = [],
+        int $status = APIContentType::STATUS_DEFINED
```

Comment:
AFAIK with `STATUS_DRAFT` this should return `ContentTypeDraft` instance, and it wont.